eventkit_cloud/ui/static/ui/app/actions/dataPackActions.js

```diff
@@ -21,19 +21,57 @@ export function getRuns(params, options = {}, isAuto) {
 
         dispatch({ type: types.FETCHING_RUNS, cancelSource: source });
 
+        const status = [];
```

Comment:
The reason for doing that was so that `getRuns()` could be reusable from within DashboardPage. It seemed like there would be a lot of complex duplicated code otherwise. I think it's also a bonus to hide the raw param generation behind more of a proper interface so that it's clear what can and can't be done without having to dig into the api.